DragonLoopApp/DragonLoopApp.iOS/CustomRenderer/CustomRenderer.cs

```diff
@@ -16,7 +16,7 @@ namespace DragonLoopApp.iOS.CustomRenderer
     public class CustomMapRenderer : MapRenderer
     {
         UIView customPinView;
-        List<CustomPin> customPins;
+        List<CustomPin> customPins = new List<CustomPin>();
```

Comment:
Is initializing this still needed even though CustomPins is getting initialized in the CustomMap constructor? If it is, you should use `var customPins = new List<CusomPin>()` to avoid the duplicate type declaration.